blueprints/eslint-config-simplabs/files/.eslintrc.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   root: true,
-  extends: 'simplabs',
+  extends: [
+    'simplabs',
+    'simplabs/rules/ember',
```

Comment:
although I'm thinking about moving those rulesets into a `plugins` folder instead of `rules`